src/main.py

```diff
@@ -46,19 +47,30 @@ def open_ticket(log: list[ChatNode], email: str) -> None:
 # Helper CLI that adds tab-completion to reply choices provided by the chatbot
 chat_cli = chatbot.Cli(chat)
 
-request = chat.START
-
-while (nodes := chat.advance(request)):
-    match nodes[0].type:
-        case "o":
-            print(f"Chatbot: {nodes[0].content}")
-        case _:
-            request = chat_cli.input("You: ")
-
-# Find the chat log in chat.log
-for node in chat.log:
-    if (node.name == "ticket_eroeffnen_email"):
-        open_ticket(chat.log, node.content)
-
-conn.commit()
-conn.close()
+if __name__ == "__main__":
+    # Main CLI for chatbot with optional debug
+    parser = argparse.ArgumentParser(description='Run chatbot with optional debug output')
+    parser.add_argument('--debug', action='store_true', help='Enable debug output of path taken')
+    args, unknown = parser.parse_known_args()
+    # Pass debug flag into environment
+    import builtins
+    builtins._CHAT_DEBUG = args.debug
+
+    request = chat.START
+
+    while (nodes := chat.advance(request)):
+        match nodes[0].type:
+            case "o":
+                print(f"Chatbot: {nodes[0].content}")
+            case _:
+                request = chat_cli.input("You: ")
+        if args.debug:
+            print("Path taken:", " -> ".join(node.name for node in chat.log))
```

Comment:
Just a stylistic comment: maybe we should print the path taken before the actual chat line (line 62 instead of 68).